### PR TITLE
Add frida-trace documentation

### PR DESCRIPTION
**Pull request by mosherubin**
This is a first stab at documenting the frida-trace tool's command line
options.  In this PR there are only seven (7) options that are described
in depth.

It is hoped that the other command line options will eventually all be
documented.

### CODE REVIEW

File: _docs/frida-trace.md [hunk 1]
```diff
@@ -7,12 +7,189 @@ permalink: /docs/frida-trace/
...
+  'frida-server-12.8.0-android-arm' binary to the Android's /data/local/tmp 
+  folder.  Using adb shell, you would run the server in the background 
+  (e.g. frida-server-12.8.0-android-arm &).</p>
+</div>
+
+## -I, -X: include/exclude module
+
+These options allow you to include or exclude **all** functions in a particular 
+module (e.g., *.so, *.dll) in one, single option.  The option expects a regular
+expression for matching one or more modules.  Any module that matches the
```
Comment (oleavr) [inline _docs/frida-trace.md]: We only support globs:
https://github.com/frida/frida-gum/blob/5858a8fecb291c3e60c300191ef156c8c4b36f77/gum/gummoduleapiresolver.c#L145
As it's built on [GLib.PatternSpec](https://developer.gnome.org/glib/stable/glib-Glob-style-pattern-matching.html).

Comment (mosherubin) [reply _docs/frida-trace.md]: Spot on comment, done.

**Comment by mosherubin** [comment]
A general comment: I documented those command line options that I used extensively and had experience with.  I have not had the opportunity to use the other options and will leave them for those who do understand and use them.  This does not preclude my tackling options, learning what they're about, and documenting them, too.

**Comment by mosherubin** [comment]
Hi oleavr,

I just added some "note-info" text explaining frida-trace's concept of a "working set", and the importance of ordering the include/exclude command line options correctly.  I remember clearly how, when I began using frida-trace, I was confounded and confused when I specified the include/exclude options as I thought was correct, only to get what seemed to be totally inconsistent results.  It was only when I delved into the code, saw the use of "exports:<module>:<function>" for reducing the key/value-based working set that I understood what needed to be done.

I hope to ease a newbie's journey by describing this up front.

**Pull request merged**